apinetlet/controllers/network_controller_test.go

```diff
@@ -53,6 +53,12 @@ var _ = Describe("NetworkController", func() {
 		}
 		Eventually(Get(apiNetNetwork)).Should(Succeed())
 
+		By("ensuring the network state is initialized")
+		Eventually(func(g Gomega) {
+			g.Expect(Get(network)).Should(Succeed())
+			g.Expect(network.Status.State).Should(BeEquivalentTo(networkingv1alpha1.NetworkStatePending))
+		})
```

Comment:
Not required for the behavioral test - we only care if the network is eventually allocated or stays in pending state.